tests/unit/test_winAPI/test_displayTracking.py

```diff
@@ -0,0 +1,81 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+
+from winAPI import displayTracking
+from winAPI.displayTracking import (
+	_updateOrientationState,
+	Orientation,
+	OrientationState,
+)
+
+
+class Test_UpdateOrientationState(unittest.TestCase):
+	def setUp(self) -> None:
+		displayTracking._orientationState = OrientationState()
+
+	def tearDown(self) -> None:
+		displayTracking._orientationState = OrientationState()
+
+	def test_landscapeInitiallySet(self):
+		monitorSet = _updateOrientationState(1, 2)
+		self.assertTrue(monitorSet)
+		self.assertEqual(
+			displayTracking._orientationState.style,
+			Orientation.LANDSCAPE
+		)
+
+	def test_portraitInitiallySet(self):
+		monitorSet = _updateOrientationState(2, 1)

```

Comment:
The `initiallySet` tests are not very clear since `_updateOrientationState` is a mutator rather than an intializer. I think it would clearer to handle all `_updateOrientationState` behavior by putting the module into a known state by modifying `_orientationState` directely then calling `_updateOrientationState` and testing the results.